operator/src/main/java/org/bf2/operator/operands/KafkaCluster.java

```diff
@@ -195,6 +194,10 @@ public Kafka kafkaFrom(ManagedKafka managedKafka, Kafka current) {
         int actualReplicas = getBrokerReplicas(managedKafka, current);
         int desiredReplicas = getBrokerReplicas(managedKafka, null);
 
+        long storagePerBroker = getPerBrokerBytes(managedKafka.getSpec().getCapacity().getMaxDataRetentionSize(), () -> this.configs.getConfig(managedKafka).getKafka().getVolumeSize(), actualReplicas);
```

Comment:
I'm curious should we use *actual*  or *desired* replicas here? 

I lean towards desired as it reduces the number of storage size changes if actual moves towards desired incrementally. 

I don't feel this needs addressing before this PR merges.

Reply:
I wondered about this myself.  I reasoned that "something" might prevent the system reaching its desired number broker, so using actual for the calculation seemed to be better.  I don't feel strongly about this.  

I propose to leave it as it is.